GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -81,7 +81,7 @@ protected override void OnRuntimeLoad(EventArgs e)
 
             _NO_TRANSLATE_To.Text = AppSettings.DefaultCloneDestinationPath;
 
-            if (CanBeGitURL(_url) || GitModule.IsValidGitWorkingDir(_url))
+            if (CanBeGitURL(_url))
```

Comment:
Whilst functionally it may be a correct change, semantically it isn't. `IsValidGitWorkingDir` has nothing to do with checking `URL`, so merging the checks alters the meaning of `CanBeGitURL`.

Reply:
You are absolutely right from a generic point of view. But git & GE do name it "URL" also if it is just a plain OS path to a repo.